README.md

```diff
@@ -1,2 +1,110 @@
-# recipe_app_rails
-Good repo
+# Blog App
```

Comment:
- [ ] Kindly update your `project title` here to make your Readme file professional.